test/e2e/yurt/yurtstaticset.go

```diff
@@ -0,0 +1,368 @@
+/*
+Copyright 2023 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yurt
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openyurtio/openyurt/pkg/apis/apps/v1alpha1"
+	"github.com/openyurtio/openyurt/test/e2e/util"
+	ycfg "github.com/openyurtio/openyurt/test/e2e/yurtconfig"
+)
+
+const (
+	staticPodPath string = "/etc/kubernetes/manifests"
+)
+
+var _ = Describe("yurtStaticSet Test", Ordered, func() {
+	ctx := context.Background()
+	timeout := 60 * time.Second
+	k8sClient := ycfg.YurtE2eCfg.RuntimeClient
+	nodeToImageMap := make(map[string]string)
+
+	var updateStrategyType string
+	var namespaceName string
+
+	yurtStaticSetName := "busybox"
+	podName := "busybox"
+	testContainerName := "bb"
+	testImg1 := "busybox"
+	testImg2 := "busybox:1.36.0"
+
+	createNamespace := func() {
+		ns := corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
```

Comment:
Is it more reasonable to use Namespace filed? (including client.ObjectKey below)
```suggestion
				Namespace: namespaceName,
```

Reply:
Sorry for not understanding your suggestion. The namespace is a cluster scope resource, and its namespace field will be ignored